app/src/main/java/org/projectbuendia/client/ui/dialogs/EditPatientDialogFragment.java

```diff
@@ -56,6 +62,8 @@
     @InjectView(R.id.patient_sex_female) RadioButton mSexFemale;
     @InjectView(R.id.patient_sex_male) RadioButton mSexMale;
 
+    private static final Pattern ID_PATTERN = Pattern.compile("([a-zA-Z]+)/?([0-9]+)");
```

Comment:
Seeing as you're relying upon the regex not to match for ids without a prefix, you might want to just go with `"([a-zA-Z]+)/([0-9]+)"` for this regex (i.e. make the `/` required).